.github/workflows/pythonpackage.yml

```diff
@@ -59,10 +59,19 @@ jobs:
 
       - name: Install PyGeoprocessing
         run: |
-            conda install --file requirements.txt
-            conda install gdal==${{ matrix.gdal }} setuptools build
+            # Installing all of these in the same line will force conda to
+            # resolve all requirements at once, reducing the chance of DLL
+            # conflicts.
+            conda install --file requirements.txt gdal==${{ matrix.gdal }} setuptools build
             python -m build --wheel
             python -m pip install $(find dist -name "*.whl")
 
+      - name: Mac dylib hack
+        if: matrix.os == 'macos-latest'
+        run: |
+            # Specific fontconfig needed for binary build issues on mac.
+            # See natcap/pygeoprocessing#252
+            conda install fontconfig=2.13.1
```

Comment:
Oofta!

Reply:
Sorry this slipped through the cracks! Now that it's been a while, do we still need this?

Reply:
To the best of my knowledge, we do not!  Removed in 8c7eaeb so let's see what happens in the tests.